tests/unit/httpd_test.cc

```diff
@@ -2314,6 +2314,10 @@ SEASTAR_TEST_CASE(test_client_close_connection) {
     });
 }
 
+SEASTAR_TEST_CASE(test_client_close_connection_content_length) {
+    return test_client_close_connection(false);
+}
```

Comment:
I just noticed that our client tests are in the file `httpd_test.cc` (where the "d" means daemon - i.e., server). That's strange.